lms/templates/main.html

```diff
@@ -68,24 +74,19 @@
     <%static:js group='base_vendor'/>
   % else:
     <%static:js group='main_vendor'/>
+    <%static:js group='application'/>
```

Comment:
Note to reviewer: had to move this up (was below at line 163) for dependencies to be loaded before RequireJS config.